access-grant/src/main/java/com/inrupt/client/accessgrant/AccessGrantClient.java

```diff
@@ -181,6 +182,41 @@ public CompletionStage<AccessGrant> issue(final URI type, final URI agent, final
         });
     }
 
+    /**
+     * Verify an access grant or request.
+     *
+     * @param accessGrant the access grant to verify
+     * @param options the options to which this credential applies
+     * @return the next stage of completion containing the resulting credential
+     */
+    public CompletionStage<VerificationResponse> verify(final AccessGrant accessGrant, final Set<String> options) {
+        return v1Metadata().thenCompose(metadata -> {
+            
+            final Map<String, Object> presentation = new HashMap<>();
+            presentation.put(VERIFIABLE_CREDENTIAL, accessGrant);
```

Comment:
The `/verify` endpoint can also take a `VerifiablePresentation`. Do you need this in this client?